GameData/KerbalismConfig/Support/RealismOverhaul.cfg

```diff
@@ -0,0 +1,40 @@
+@PART[*]:HAS[#engineType[NERVA,NERVAII]]:AFTER[RealismOverhaulEngines]
+{
+	MODULE
+	{
+		name = ProcessController
+		resource = _TurbineGenerator
+		title = Turbine Generator
+		capacity = #$../MODULE[ModuleResourceConverter]/OUTPUT_RESOURCE:HAS[#ResourceName[ElectricCharge]]/Ratio$
+		running = true
+		toggle = true
+	}
+	!MODULE[ModuleResourceConverter] {} // replace stock converter module
+	MODULE
+	{
+		name = Emitter
+		radiation = 0.00055555 // 2 rad/h
+	}
+}
+@PART[*]:HAS[#engineType[BNTR]]:AFTER[RealismOverhaulEngines]
+{
+	MODULE
+	{
+		name = ProcessController
+		resource = _BraytonGenerator
+		title = Brayton Generator
+		capacity = #$../MODULE[ModuleResourceConverter]/OUTPUT_RESOURCE:HAS[#ResourceName[ElectricCharge]]/Ratio$
+		// capacity=1 == 25kW, but Ratio=1 == 1kW.
+		// FIXME: would be nice to fish the 25 out of the Process definition's "output = ElectricCharge@25"
```

Comment:
Would you prefer to normalize the output of the process then?

Reply:
Not knowing why it was 25 in the first place, no opinion. I was confused for a while because the value set in RO is _also_ 25; for all I know of the topic, brayton generators might be inherently stuck at multiples of 25 and that capacity makes sense.

Reply:
I see, there is no real reason for 25 to be the default, so you can go ahead and fix it if you want